enricher/ffa/filterknownbinaries/dpkg.go

```diff
@@ -0,0 +1,95 @@
+package filterknownbinaries
+
+import (
+	"bufio"
+	"context"
+	"errors"
+	"io"
+	"path"
+	"strings"
+
+	"github.com/google/osv-scalibr/artifact/image/layerscanning/image"
+	scalibrfs "github.com/google/osv-scalibr/fs"
+	"github.com/google/osv-scalibr/fs/diriterate"
+)
+
+var (
+	dpkgInfoDirPath  = "var/lib/dpkg/info"
+	ignorePathPrefix = []string{
+		"/var/lib/dpkg/info",
```

Comment:
Not sure if I understand what this is for. Are there binaries under /var/lib/dpkg/info? Do we want to exclude them? Does the code that reads this store paths with a leading "/"?

Reply:
> Are there binaries under /var/lib/dpkg/info?
Yes, pre/post install scripts, and pre/post removal scripts
> Do we want to exclude them?
Yes, it's very noisy and is not covered by list files.
>  Does the code that reads this store paths with a leading "/"?
Hmm... now looking at the code it seems like it doesn't, but I was pretty sure when I tested it, it worked. I'll update it and test it a bit more.

Reply:
Thanks. It'd also be great if we could have a comment about why things under /var/lib/dpkg/info are excluded (basically what you explained in this thread).